### PR TITLE
Fix config key naming host =>hostname

### PR DESCRIPTION
**Pull request by BorisOsipov**
## Proposed changes

[//]: # (Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.)

## Types of changes

[//]: # (What types of changes does your code introduce to WebdriverIO?)
[//]: # (_Put an `x` in the boxes that apply_)

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

[//]: # (_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._)

- [ ] I have read the [CONTRIBUTING](https://github.com/webdriverio/webdriverio/blob/master/CONTRIBUTING.md) doc
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

## Further comments

[//]: # (If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...)

### CODE REVIEW

File: packages/wdio-sauce-service/src/launcher.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ export default class SauceLauncher {
         }, config.sauceConnectOpts)
 
         config.protocol = 'http'
-        config.host = 'localhost'
```
Comment (BorisOsipov) [inline packages/wdio-sauce-service/src/launcher.js]: @christian-bromann pay attention. 
I don't know very well how `wdio-sauce-service` and why we need it here

**Comment by BorisOsipov** [comment]
Awesome to have unit test 😊

**Comment by abjerstedt** [comment]
Isn't this going to be a breaking change for people currently using that property?

**Comment by BorisOsipov** [comment]
@abjerstedt yes but currently we have inconsistency caused by
https://github.com/webdriverio/webdriverio/pull/3260
https://github.com/webdriverio/webdriverio/pull/3211/files

**Comment by wswebcreation** [comment]
@abjerstedt 

You are right, this is a breaking change and costed me a few minutes to debug 😢 , updated the appium-boilerplate to

**Pull request merged**